doc/source/cluster/kubernetes/k8s-ecosystem/scheduler-plugins.md

```diff
@@ -0,0 +1,55 @@
+# KubeRay integration with scheduler plugins
+
+The [kubernetes-sigs/scheduler-plugins](https://github.com/kubernetes-sigs/scheduler-plugins) repository provides out-of-tree scheduler plugins based on the scheduler framework.
+
+Starting with KubeRay v1.4.0, KubeRay integrates with the [PodGroup API](https://github.com/kubernetes-sigs/scheduler-plugins/blob/93126eabdf526010bf697d5963d849eab7e8e898/site/content/en/docs/plugins/coscheduling.md) provided by scheduler plugins to support gang scheduling for RayCluster custom resources.
+
+## Step 1: Create a Kubernetes cluster with Kind
+
+```sh
+kind create cluster --image=kindest/node:v1.26.0
+```
+
+## Step 2: Install Scheduler Plugins
+
+Follow [install.md](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/doc/install.md) in the scheduler-plugins repository to install the scheduler plugins.
```

Comment:
Is it better to reference this site?

https://scheduler-plugins.sigs.k8s.io/docs/user-guide/installation/